docs/command-reference.md

```diff
@@ -0,0 +1,632 @@
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container delete (rm)`
+
+Removes one or more containers. If the container is running, you may force deletion with `--force`. Without a container ID, nothing happens unless `--all` is supplied.
+
+**Usage**
+
+```bash
+container delete [OPTIONS] [CONTAINER...]
+```
+
+**Options**
+
+*   `-f, --force`: remove running containers by sending SIGKILL
+*   `-a, --all`: remove all containers
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container list (ls)`
+
+Lists containers. By default only running containers are shown. Output can be formatted as a table or JSON.
+
+**Usage**
+
+```bash
+container list [OPTIONS]
+```
+
+**Options**
+
+*   `-a, --all`: include stopped containers
+*   `-q, --quiet`: display only container IDs
+*   `--format <format>`: output format: `table` or `json` (default `table`)
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container exec`
+
+Executes a command inside a running container. It uses the same process flags as `container run` to control environment, user, and TTY settings.
+
+**Usage**
+
+```bash
+container exec [OPTIONS] CONTAINER COMMAND [ARG...]
+```
+
+**Key flags**
+
+*   `-e, --env <key=value>`: set environment variables inside the exec session
+*   `--env-file <file>`: read environment variables from a file
+*   `-i, --interactive`: keep STDIN open
+*   `-t, --tty`: allocate a TTY
+*   `-u, --user <user>`: run as a specific user
+*   `--uid <uid>`, `--gid <gid>`: specify numeric UID/GID
+*   `--cwd <path>`: set working directory
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container logs`
+
+Fetches logs from a container. You can tail the logs (`-f`/`--follow`), restrict the number of lines shown, or view boot logs.
+
+**Usage**
+
+```bash
+container logs [OPTIONS] CONTAINER
+```
+
+**Options**
+
+*   `-f, --follow`: follow the log output for real-time streaming
+*   `--boot`: show the container's boot logs instead of STDOUT/STDERR
+*   `-n <lines>`: number of lines from the end of logs to display
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container inspect`
+
+Displays detailed container information in JSON. Pass one or more container IDs to inspect multiple containers.
+
+**Usage**
+
+```bash
+container inspect [OPTIONS] CONTAINER...
+```
+
+No additional flags; uses global flags for debug and help.
+
+## Image Management
+
+### `container image list (ls)`
+
+Lists local images. Verbose output provides additional details such as image ID, creation time and size; JSON output provides the same data in machine-readable form.
+
+**Usage**
+
+```bash
+container image list [OPTIONS]
+```
+
+**Options**
+
+*   `-q, --quiet`: show only image names
+*   `-v, --verbose`: produce verbose listing
+*   `--format <format>`: `table` (default) or `json`
+*   **Global**: `--debug`, `-h`/`--help`
+
+### `container image pull`
+
+Pulls an image from a registry. Supports specifying a platform and controlling progress display.
```

Comment:
I'm not sure where we landed with whether or not we want to pull all platforms by default if no platform is specified. But should we include that default info here?